src/parser/parse_redirect_utils.c

```diff
@@ -0,0 +1,47 @@
+#include <commands/commands.h>
+#include <parser/command_table.h>
+#include <executor/executor_utils.h>
+
+/*
+	This'll probably change, saving for syntax checker
+*/
+void	check_cmd_str_validity(char *cmd_str)
+{
+	int	i;
+
+	i = 0;
+	while (cmd_str && cmd_str[i])
+	{
+		if (cmd_str[i] == DOUBLE_QUOTES)
+		{
+			i++;
+			while (cmd_str[i] && cmd_str[i] != DOUBLE_QUOTES)
+				i++;
```

Comment:
I think we can do:
```C
while (*cmd_str)
{
    if (*cmd_str == DOUBLE_QUOTES)
    {
      ++cmd_str;
      cmd_str = ft_strcchar(&cmd_str[i], DOUBLE_QUOTES);
      if (!cmd_str)
            handle_errors(16, "Syntax error in checker for outfile");
      }
      if (!is_valid_redirection(cmd_str))
              handle_errors(16, "Syntax error in checker for outfile");
      ++cmd_str;
}
```

something like this.

Reply:
Will think of a cleaner way to check this, specifically checking with `i` because the first char can also be a valid right_angled bracket with being preceded by spaces...